SEP/SEP-0003/sep-0003.md

```diff
@@ -0,0 +1,44 @@
+# Property paths with a min/max hop
+
+* Short Name: min-max-propery-path
+* SEP Number: [SEP-0003](sep-0003.md)
+* Author: [Jerven Bolleman](https://github.com/jervenbolleman)
+
+## Abstract
+
+In the SPARQL 1.1. working group, a form of bounded property path where supported in the late [drafts](https://www.w3.org/TR/2012/WD-sparql11-query-20120105/#propertypaths). The form being a path element _elt_, followed by a curly bracket enclosed min/max pattern.
+
+||Syntax Form||Matches||
+| _elt_{n,m} | A path of between n and m occurrences of elt. |
+| _elt_{n}   | A path of exactly n occurrences of elt. |
+| _elt_{n,}  | A path of n or more occurrences of elt. |
+| _elt_{,n}  | A path of between 0 and n occurrences of elt. |
```

Comment:
Just wondering if would be valuable to have this mean "_A path of between 1 and n occurrences of elt_" instead?
I've noticed that an implementation of the zero-case can be tricky to optimize, while users may not really need it this often(?).

Reply:
I tend to agree that the zero-length case is often not what is wanted (especially in cases where both subject and object are variables), and requiring the full `{0,m}` form to get those semantics would probably be desirable.

Reply:
Imho, it would be a bit strange to have `{,n}` mean `{1,n}` and not `{0,n}` because 1 is not the lower bound of the possible values. What about not allowing `{,n}`? `{1,n}` and `{0,n}` are just one key press away from `{,n}`.

Reply:
@Tpt on reflection, I think being explicit is better than being implicit so disallowing the form `{,n}` seems sensible. Adding `{n,*}` to mean at least n hops seems sensible to me.

Reply:
Great! My personal opinion would be to keep `{n,}` instead of `{n,*}` in order to avoid adding an extra symbol and use the same syntax as a lot of regex languages.

Reply:
Could we have discussions about content on issue #101?

This keeps things together. We can merge draft SEPs and PRs to SEPs without losing important input.